libledger/LedgerParam.h

```diff
@@ -68,6 +68,9 @@ struct ConsensusParam
     // enable optimize ttl or not
     bool enableTTLOptimize;
     bool enablePrepareWithTxsHash;
+
+    bool broadcastPrepareByTree;

```

Comment:
Make sure this option is only useable for rPBFT

Reply:
The `broadcastPrepareByTree` configuration is only used after init `RotatingPBFTEngine`, so this option is only used by `rPBFT`.